webapp/components/admin_console/log_settings.jsx

```diff
@@ -149,7 +149,7 @@ export default class LogSettings extends AdminSettings {
                     helpText={
                         <FormattedMessage
                             id='admin.log.locationDescription'
-                            defaultMessage='File to which log files are written. If blank, will be set to ./logs/mattermost, which writes logs to mattermost.log. Log rotation is enabled and every 10,000 lines of log information is written to new files stored in the same directory, for example mattermost.2015-09-23.001, mattermost.2015-09-23.002, and so forth.'
+                            defaultMessage='Path to which log files are written. If blank, will be set to ./logs/, which writes logs to mattermost.log. Log rotation is enabled and every 10,000 lines of log information is written to new files stored in the same directory, for example mattermost.2015-09-23.001, mattermost.2015-09-23.002, and so forth.'
```

Comment:
This might be better:

*The location of the log files. If blank, they are stored in the ./logs/ directory. Mattermost creates the directory if it does not exist.*

Is it true that the directory is created if it doesn't already exist?